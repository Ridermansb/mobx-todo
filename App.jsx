import React  from 'react'

const App = (props) => <span>oi</span>

export default App;