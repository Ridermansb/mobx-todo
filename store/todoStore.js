import { action, observable } from 'mobx'

class TodoStore {
  @observable todos = []
  @action add(todo) {
    this.todos.push(todo);
  }
}

export { TodoStore };
export default new TodoStore();