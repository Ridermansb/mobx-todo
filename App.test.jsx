import React from 'react';
import App from './App';
import renderer from 'react-test-renderer';
import todoStore from './store/todoStore'

describe('My App component', () => {
  test('renders without crashing', () => {
    const wrapper = renderer.create(<App todoStore={todoStore} />);
    expect(wrapper.toJSON()).toMatchSnapshot();
  });
});