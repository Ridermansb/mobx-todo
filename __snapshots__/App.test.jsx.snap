// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`My App component renders without crashing 1`] = `
<span>
  oi
</span>
`;
